create_reference_tsv.py

```diff
@@ -0,0 +1,38 @@
+import sys
+inFile = sys.argv[1]
+outFile = sys.argv[2]
+
+sampleDict = {}
+
+with open(inFile, 'r') as f:
+	lines = f.readlines()
+
+k = 0
+for i in lines:
+	sample = i.split('/')[-1].split('_')[0]
+	readnr = i.split('/')[-1].split('_')[3]
```

Comment:
I think these assignments are pretty vulnerable. If a sample name contains one or more underscores, the logic won't work.
A more reliable approach would be to use a regexp, e.g. something like: 
```suggestion
        import re
	pattern = r'^(.*)_(S\d+)_L(\d+)_(R\d)_001.*$'
	m = re.match(os.path.basename(i), pattern)
	if m:
	  sample = m.group(1)
	  ssheet_idx = m.group(2)
	  laneno = int(m.group(3))
	  readnr = m.group(4)
	else:
	  continue
```

Reply:
@b97pla As it is now underscores are not allowed in sample name because of a "feature" in bcl2fastq. However, it might be worth changing anyway to make the script more secure.

Reply:
ah ok, yes I would go for the regex approach regardless